intercept_anthropic_messages_blocking.go

```diff
@@ -77,7 +87,8 @@ func (i *AnthropicMessagesBlockingInterception) ProcessRequest(w http.ResponseWr
 	var cumulativeUsage anthropic.Usage
 
 	for {
-		resp, err = svc.New(ctx, messages)
+		// TODO add outer loop span (https://github.com/coder/aibridge/issues/67)
+		resp, err = i.traceNewMessage(ctx, svc, messages) // traces svc.New(ctx, msgParams) call
```

Comment:
We discussed renaming this method; I see you fixed it elsewhere but not here - you probably just overlooked it.
I'd like the method to be named by what it does, not what we do around what it does.